### ISSUE TITLE
The '--help' flag for truss should give directions

### ISSUE THREAD

**Issue by hasLeland** [metaverse/truss#34]
Currently, the output of truss is quite bare, and doesn't include at all how you'd run this on a set of proto files, nor does it include what truss is, how to find more information. Currently it looks like this:

```
Usage of truss:
  -nobuild
        Set -nobuild to generate code without building
```

And it should include things like:
- How to pass in proto files
- What `truss` is meant to do
- Where to find more about `truss` (a link back to this github, for example)

**Comment by hasAdamr** [comment]
This was mostly fixed in pull #70. Not sure if we need to add a link back to github. Adding a little description could be good though. 

A quick idea to put up.

```
truss generates go-kit services using protobuf and grpc definitions.
```

**Comment by zaquestion** [comment]
For reference, help is currently
```
$ truss --help
Usage: truss [options] <protofile>...
      --pbout string    Go package path where the protoc-gen-go .pb.go files will be written
      --svcout string   Go package path where the generated Go service will be written
  -v, --verbose         verbose output
```
